tests/testthat/setup-workflow-ref.R

```diff
@@ -170,3 +170,15 @@ cleanup <- function() {
   suppressSpecificWarning(rm(log_df, pos = parent.frame()),.regexpr = "object.+not found")
 }
 
+#' Temporarily perturb a file
+#'
+#' Appends a line to the end of a text file, and flexibly revert the change.
```

Comment:
this says `flexibly revert`. I'm not sure what that means, but I read it as `optionally revert` which doesn't seem to be true. If I'm misunderstanding just ignore.

Reply:
By "flexibly," I mean that the caller can control _when_ the change is reverted through the `envir` argument. I can spell that out if you prefer, e.g., "and revert the change in an environment specified by the caller."

Reply:
Yea that's more clear. Thanks.